### DISCUSSION TITLE
Watchtower never finding any new images?

### DISCUSSION THREAD

**Discussion by realdannys** [containrrr/watchtower#855] [Q&A] [answered]
I've installed Watchtower on my QNAP Nas. I believe I've set everything up correct, it launches and it sits there saying its checking but it's never found any updated images.

I've just got 10,000 lines of
```
DEBU[461703] Scheduled next run: 2021-03-22 10:03:53 +0000 UTC 
DEBU[462003] Checking containers for updated images       
DEBU[462003] Retrieving running containers                
DEBU[462003] Scheduled next run: 2021-03-22 10:08:53 +0000 UTC 
DEBU[462303] Checking containers for updated images       
DEBU[462303] Retrieving running containers                
DEBU[462303] Scheduled next run: 2021-03-22 10:13:53 +0000 UTC 
DEBU[462603] Checking containers for updated images       
DEBU[462603] Retrieving running containers      
```

My set up is as follows

`CMD --debug true --cleanup true`

I've got my PGID and PUID set to 0 and 0 which is the admin on the system

`WATCHTOWER_MONITOR_ONLY: TRUE`

`/var/run/docker.sock:/var/run/docker.sock`

and network is set as host.

any ideas? Is the Qnap docker.sock running somewhere else?

**Comment by simonaronsson** [comment, marked as answer]
It's the `true` part of your command that is causing it. With the current config, watchtower will exclusively look for containers named `true`. The flags themselves take no argument, so `CMD --debug --cleanup` should do.

**Reply by realdannys** [reply]
Ahh ok, thank you!

**Reply by realdannys** [reply]
It did fire up and it updated everything though I had WATCHTOWER_MONTOR_ONLY set to true.

Was it because it was set to TRUE instead of 'true' ? I have it as this in Portainer, should this work correctly?

<img width="1302" alt="CleanShot 2021-03-22 at 15 31 50" src="https://user-images.githubusercontent.com/3345471/112018209-4524f480-8b26-11eb-9592-4d66189dea1e.png">

**Reply by simonaronsson** [reply]
Did it restart the containers as well? Monitor only might be a bit of a confusing name for that option, as it will still pull all the images, although it shouldn't recreate any containers.

It looks like portainer wraps the value in a string, which could explain why it's not applied. Since you're already running a custom CMD, maybe the easiest solution would be to add the `--monitor-only` flag instead?
